Makefile

```diff
@@ -0,0 +1,119 @@
+# Makefile for Ansible Chatbot Stack
+
+# Default values for environment variables
+QUAY_ORG ?= $(QUAY_ORG)
```

Comment:
Out of curiosity; why do some variables use `$(...)` (round brackets) and others `${...}` (squiggly brackets)?

Reply:
No reason, bad habit of using curly braces for env vars 😅, instead of using Makefile norm of '()'. Updated to round brackets.